test/fixtures_s3_test.go

```diff
@@ -0,0 +1,153 @@
+//+build integration fixtures
+
+package test_test
+
+import (
+	"fmt"
+	"github.com/applike/gosoline/pkg/blob"
+	"github.com/applike/gosoline/pkg/fixtures"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/applike/gosoline/pkg/test/suite"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"io/ioutil"
+	"testing"
+)
+
+const (
+	basePath      = "test_data/s3_fixtures_test_data"
+	basePathPurge = "test_data/s3_fixtures_purge_test_data"
+	configName    = "test"
+)
+
+type FixturesS3Suite struct {
+	suite.Suite
+	logger     mon.Logger
+	bucketName string
+	client     s3iface.S3API
+	loader     fixtures.FixtureLoader
+}
+
+func (s *FixturesS3Suite) SetupSuite() []suite.Option {
+	return []suite.Option{
+		suite.WithLogLevel("debug"),
```

Comment:
not needed, is it?

Reply:
Yeah I added that to debug. Now it's been removed.